From 1222e48fd155300a16376fbefb33fb2e338f5c79 Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Mon, 2 May 2016 16:30:37 -0400
Subject: [PATCH 20/33] net: netcp: cpts: update to 64bit for 4.4.y

The 4.4.y kernel has changed the 32bit variables to 64bit that
requires API change from ns_to_timespec() to ns_to_timespec64().

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
---
 drivers/net/ethernet/ti/netcp_cpts.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/ti/netcp_cpts.c b/drivers/net/ethernet/ti/netcp_cpts.c
index 415fa3d..5544361 100644
--- a/drivers/net/ethernet/ti/netcp_cpts.c
+++ b/drivers/net/ethernet/ti/netcp_cpts.c
@@ -469,7 +469,7 @@ static int cpts_ts_comp_add_reload(struct cpts *cpts, s64 add_ns, int enable)
 			/* report the event */
 			ns = cpts_tstamp_cyc2time(cpts, event->low);
 			pevent.type = PTP_CLOCK_PPSUSR;
-			pevent.pps_times.ts_real = ns_to_timespec(ns);
+			pevent.pps_times.ts_real = ns_to_timespec64(ns);
 			ptp_clock_event(cpts->clock, &pevent);
 			reported = 1;
 
@@ -511,7 +511,7 @@ static int cpts_ts_comp_hw_ts_ev_report_restart(struct cpts *cpts)
 			/* report the event */
 			ns = cpts_tstamp_cyc2time(cpts, event->low);
 			pevent.type = PTP_CLOCK_PPSUSR;
-			pevent.pps_times.ts_real = ns_to_timespec(ns);
+			pevent.pps_times.ts_real = ns_to_timespec64(ns);
 			ptp_clock_event(cpts->clock, &pevent);
 			++reported;
 
-- 
1.9.1

